src/main/java/li/cil/tis3d/common/tileentity/ComputerTileEntity.java

```diff
@@ -151,12 +151,11 @@ public void load(final CompoundTag tag) {
         loadServer(tag);
     }
 
-    @Override
-    public CompoundTag save(final CompoundTag compound) {
-        final CompoundTag tag = super.save(compound);
-        saveServer(tag);
-        return tag;
-    }
+	@Override
+	protected void saveAdditional(CompoundTag nbt) {
+        saveServer(nbt);
```

Comment:
Indentation seems off here; also, any reason the call to super comes second and not as the first thing in the override?

Reply:
looks like the thing i copied from had tabs instead of spaces. terrible, evil. i'm correcting on my end.

there are three reasons why they're in the order they're in:
a) that's how everyone else does it, and i'mma jump off bridges where previous successful modders went when i don't see another path
b) during my frustrated flailing i thought i tried it in the other order and it didn't seem to work
c) i don't actually know what i'm doing.

but i just tested it in that order, and yeah, seems to work fine, so i'm making the change.

Reply:
Cool, I just usually like to call the base method first, because I find the code-flow to be more easy to understand that way.